TestFoundation/TestNotificationQueue.swift

```diff
@@ -117,7 +117,7 @@ FIXME SR-4280 timeouts in TestNSNotificationQueue tests
             // post 2 notifications for the NSDefaultRunLoopMode mode
             queue.enqueue(notification, postingStyle: .now, coalesceMask: [], forModes: [runLoopMode])
             queue.enqueue(notification, postingStyle: .now)
-            // here we post notification for the NSRunLoopCommonModes. It shouldn't have any affect, because the timer is scheduled in NSDefaultRunLoopMode.
+            // here we post notification for the NSRunLoop.commonModes. It shouldn't have any affect, because the timer is scheduled in NSDefaultRunLoopMode.
```

Comment:
> NSRunLoop.commonModes

This should be `RunLoopMode.commonModes`.

https://developer.apple.com/documentation/foundation/runloopmode/1408609-commonmodes